src/main/resources/public/src/views/ProcessingStat.vue

```diff
@@ -124,7 +133,7 @@
         </div>
         <div class="form-group row" v-if="organizationSearch !== ''">
             <label for="organization-model-list" class="col-sm-3 col-form-label col-form-label-sm">
-                Models processed for selected Organization:
+                {{ $t('models_processed_for_selected_organization') }}
             </label>
             <select id="organization-model-list" class="form-control-sm" v-model="selectedModelForOrganization">
                 <option>All</option>
```

Comment:
You forgot this "All"

Reply:
No, I left it voluntarily because the operation of the selection is bad